src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1356,32 +1539,54 @@ def forward(
         )
         return_dict = return_dict if return_dict is not None else self.config.return_dict
 
-        # Embed images and text queries
-        outputs = self.image_text_embedder(
-            input_ids=input_ids,
-            pixel_values=pixel_values,
-            attention_mask=attention_mask,
-            output_attentions=output_attentions,
-            output_hidden_states=output_hidden_states,
-        )
+        query_embeds = None
+        query_mask = None
+        assert (
```

Comment:
No new asser in the code base, please use a test and raise an approriate exception.